content/projects/nodejs/ajax/_index.md

```diff
@@ -51,7 +56,7 @@ Update the "thank you page" so that it only displays for 3 seconds and then redi
 
 ## Up for a challenge?
 
-So far, you've made use of only 2 API end points(`listAllVisitors` and `deleteVisitor`). Can you figure out how to make use the `updateVisitor` endpoint? The implementation here is completely up to you.
+So far, you've made use of only 3 API end points(`POST`, `GET` and `DELETE`). Can you figure out how to make use the `PUT` endpoint? The implementation here is completely up to you.
```

Comment:
for grammar i will rephrase to"So far, you've utilized only three API endpoints (POST, GET, and DELETE). Can you figure out how to use the PUT endpoint? The implementation here is entirely up to you"